projects/layout/src/modules/box/box.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'sky-box',
+  templateUrl: './box.component.html',
+  styleUrls: ['./box.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SkyBoxComponent {
+
+  /**
+   * Specifies an ARIA label for the box component
+   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * If the box component includes a visible label, use `ariaLabelledBy` instead.
+   */
+  @Input()
+  public ariaLabel: string;
+
+  /**
+   * Specifies the HTML element ID (without the leading `#`) of the element that labels
+   * the box component [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * If the box component does not include a visible label, use `ariaLabel` instead.
+   */
+  @Input()
+  public ariaLabelledBy: string;
+
+  /**
+   * Specifies an ARIA role for the box component [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility) by indicating what the box component contains. For information about how an ARIA role indicates what an item represents on a web page, see the [WAI-ARIA roles model](https://www.w3.org/WAI/PF/aria/roles).
+   */
+  @Input()
+  public ariaRole: string;
```

Comment:
☝️ Since we allow consumers to put anything in the template, we should probably also make these three aria attributes available so consumers have the power to make their boxes more accessible: https://www.w3.org/WAI/GL/wiki/Using_HTML5_section_element

Reply:
Since the ARIA attributes are only being added to the `section` element in the component's HTML, why don't we just let the consumer add the `attr.aria-` values to the `sky-box` element directly? Seems like we're creating a "pass-through" API out of convenience, instead of a necessity?

Reply:
(I might be missing a use-case you're thinking of.)

Reply:
It's a good point. My thought here was driven by two things:
1. Sections have an[ implicit role of "region"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section), which is pretty generic, and need more explanation.
2. Having these inputs as a part of our public API exposes their importance to developers. It provides a spot in our docs to present the a11y issues and give a way to solve them.

Thoughts?

Reply:
Your second point is convincing. 👍